cocotb/decorators.py

```diff
@@ -85,12 +85,18 @@ class RunningTask(object):
         task.kill() will destroy a coroutine instance (and cause any Join
         triggers to fire.
     """
+
```

Comment:
Unless this is deliberate:
```suggestion
```

Reply:
It was deliberate. I know it is errant, but my IDE is highlighting the line (and a bunch of others) and it's starting to bother me.